.circleci/config.yml

```diff
@@ -239,6 +242,9 @@ commands:
             echo 'export AWS_CRYPTO_CONTEXT_STAGE=$AWS_CRYPTO_CONTEXT_STAGE_DEV' >> $BASH_ENV
             echo 'export AWS_CRYPTO_CONTEXT_PURPOSE=$AWS_CRYPTO_CONTEXT_PURPOSE' >> $BASH_ENV
             echo 'export AWS_CRYPTO_CONTEXT_ORIGIN=$AWS_CRYPTO_CONTEXT_ORIGIN' >> $BASH_ENV
+            echo 'export USE_BASIC_AUTH=$USE_BASIC_AUTH_CYPRESS' >> $BASH_ENV
+            echo 'export BASIC_AUTH_USERNAME=$BASIC_AUTH_USERNAME_DEV' >> $BASH_ENV
+            echo 'export BASIC_AUTH_PASSWORD=$BASIC_AUTH_PASSWORD_DEV' >> $BASH_ENV
```

Comment:
If you follow my advice re opting-in, this should not be necessary at all to set for cypress